regress.sh

```diff
@@ -1,18 +1,25 @@
+# upgrade pip
+python -m pip install --upgrade pip
+
 # install HardFloat
-wget -O install_hardfloat.sh https://git.io/JJ5YF
+curl -L https://git.io/JJ5YF > install_hardfloat.sh
 source install_hardfloat.sh
 
+# install various python dependencies
+pip install wheel
+pip install pytest pytest-cov
+
 # install msdsl
+pip install wheel
 pip install -e .
 
-# install a specific version of pysmt to avoid cluttering the output with warnings
+# install magma ecosystem
+# a specific version of pysmt is used to avoid cluttering the output with warnings
 pip install pysmt==0.9.0
-
-# install testing dependencies
-pip install pytest pytest-cov fault==3.0.36 magma-lang==2.1.17 coreir==2.0.120 mantle==2.0.10 hwtypes==1.4.3 ast_tools==0.0.30 kratos==0.0.31.1
+pip install fault==3.0.36 magma-lang==2.1.17 coreir==2.0.120 mantle==2.0.10 hwtypes==1.4.3 ast_tools==0.0.30 kratos==0.0.31.1
 
 # run tests
 pytest --cov-report=xml --cov=msdsl tests/ -v -r s
 
-# upload coverage
-bash <(curl -s https://codecov.io/bash)
+# upload coverage information
+curl -s https://codecov.io/bash | bash
```

Comment:
This is another Windows-related oddity -- for some reason this slight variation of CodeCov's invocation works, but the old style does not.  It seems this is a bug in the way GitHub Actions has set up the Windows machine.  Fortunately, the new way of invoking CodeCov also works on macOS and Linux.